assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -35,13 +35,20 @@ const FETCH_ACCOUNTS_PROPERTIES_PROFILES = 'FETCH_ACCOUNTS_PROPERTIES_PROFILES';
 const START_FETCH_ACCOUNTS_PROPERTIES_PROFILES = 'START_FETCH_ACCOUNTS_PROPERTIES_PROFILES';
 const FINISH_FETCH_ACCOUNTS_PROPERTIES_PROFILES = 'FINISH_FETCH_ACCOUNTS_PROPERTIES_PROFILES';
 const CATCH_FETCH_ACCOUNTS_PROPERTIES_PROFILES = 'CATCH_FETCH_ACCOUNTS_PROPERTIES_PROFILES';
+const FETCH_CREATE_ACCOUNT = 'FETCH_CREATE_ACCOUNT';
+const RECEIVE_CREATE_ACCOUNT = 'RECEIVE_CREATE_ACCOUNT';
+const START_FETCH_CREATE_ACCOUNT = 'START_FETCH_CREATE_ACCOUNT';
+const FINISH_FETCH_CREATE_ACCOUNT = 'FINISH_FETCH_CREATE_ACCOUNT';
+const CATCH_FETCH_CREATE_ACCOUNT = 'CATCH_FETCH_CREATE_ACCOUNT';
 
 const RECEIVE_ACCOUNTS = 'RECEIVE_ACCOUNTS';
 const RESET_ACCOUNTS = 'RESET_ACCOUNTS';
 
 export const INITIAL_STATE = {
 	accounts: undefined,
 	isFetchingAccountsPropertiesProfiles: false,
+	isFetchingCreateAccount: false,
+	accountTicketTermsOfServiceURL: false,
```

Comment:
This should be `undefined` initially. Any values not loaded should use `undefined`, the only exception from that are the flags for e.g. requests being in progress, which can just be booleans (like the two above).

Reply:
Set to undefined in https://github.com/google/site-kit-wp/pull/1390/commits/1e50efb5cb1acd850de1dae2cd14cb3d0dc60dcf.